src/vorta/application.py

```diff
@@ -23,7 +23,12 @@
 
 logger = logging.getLogger(__name__)
 
-APP_ID = TEMP_DIR / "socket"
+# If config.TEMP_DIR is None, this means that the app got launched withoug going through main
+# (for example when running tests or when running on schedule.)
+# In any case, if the config dirs aren't set, assume you're runnning system wide initialize appropriately
+if not config.TEMP_DIR:
+    config.init_from_platformdirs()
```

Comment:
I might be better to call `init_from_platformdirs` in `config` (every time) and override it in `__main__` when needed.

Reply:
You mean calling `init_from_platformdirs` in `application`? Or calling that method in `config`  so that it runs every time the module is loaded? Wouldn't the latter risk overwriting the variables from time to time as a side effect?

Reply:
> Or calling that method in config so that it runs every time the module is loaded?

That's what I was thinking of since it ensures the config values contain valid values.

> Wouldn't the latter risk overwriting the variables from time to time as a side effect?

Python modules are only loaded once (and cached) in a run afaik.

Reply:
Ohhh. I see. Thanks! I'll do that.